todo-list-server/.jpb/persistence-units.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PersistenceUnitSettings">
+    <persistence-units>
+      <persistence-unit name="Default">
+        <packages>
+          <package value="ru.hh.school.todolistserver" />
+        </packages>
+      </persistence-unit>
+    </persistence-units>
+  </component>
+</project>
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline